cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -143,7 +154,10 @@ func (w *wizard) activateOrchestrator() {
 		}
 	}
 
-	httpPostWithParams(fmt.Sprintf("http://%v:%v/activateOrchestrator", w.host, w.httpPort), val)
+	result := httpPostWithParams(fmt.Sprintf("http://%v:%v/activateOrchestrator", w.host, w.httpPort), val)
+	if result == "" {
```

Comment:
This doesn't give any feedback to the user in case of error. Moreover, it will mislead user in most error cases.

```suggestion
	if result == "" || result != "success" {
	fmt.Printf("Error activating orchestrator: %s\n", result)
```

Reply:
Not all endpoints necessarily return "success", e.g. `/vote` returns the transaction hash of the vote as a hexstring upon success. 

We currently don't propagate errors from `httpPostWithParams` back to the caller

Reply:
This if is for result of `/activateOrchestrator`, which is returning `success`

Reply:
fixed in cd54636